website/blog/2025-03-05-release-1.17.md

```diff
@@ -0,0 +1,438 @@
+
+Ever wanted to start a container effortlessly? Now, just enter the image (local or remote), select it from a dropdown, and with a single click, you'll get the perfect startup prompt. In no time, your container is swimming happily in its new environment!
+
+<ThemedImage
+alt="Starting a container from scratch"
+sources={{
+    light: require('./img/podman-desktop-release-1.17/start-container-from-image-light.png').default,
+    dark: require('./img/podman-desktop-release-1.17/start-container-from-image-dark.png').default,
+  }}
+/>
+
+### Mirroring of Registries – No More Fishy Workarounds
+
+When creating a Podman machine from Podman Desktop, we now include a playbook that mounts the host’s `registries.conf` into the VM at `/etc/containers/registries.conf.d/`. To handle this configuration, simply click on 'More Actions' in the Podman machine menu, and you're all set. No more struggling like a seal stuck on an ice floe.
+
+<ThemedImage
+alt="Mirroring of Registries"
+sources={{
+    light: require('./img/podman-desktop-release-1.17/setup-mirror-registry-light.png').default,
+    dark: require('./img/podman-desktop-release-1.17/setup-mirror-registry-dark.png').default,
+  }}
+/>
+
+<ThemedImage
+alt="Mirroring of Registries step 2"
+sources={{
+    light: require('./img/podman-desktop-release-1.17/setup-mirror-registry-step2-light.png').default,
+    dark: require('./img/podman-desktop-release-1.17/setup-mirror-registry-step2-dark.png').default,
+  }}
+/>
+
+### Podman 5.4
+
+A new version of Podman is here. Check out the full details: [Podman 5.4.0 Release Notes](https://github.com/containers/podman/releases/tag/v5.4.0).  
+⚠️ breaking change
+As mentioned in the podman 5.4 release notes, support for Mac/Intel is now on a best-effort basis.  
+📝 Note for 2020 Mac/Intel Users
+There is an open issue where Podman Machine isn’t booting on certain setups. It’s currently under investigation, and you can track the progress here: [Issue #25121](https://github.com/containers/podman/issues/25121).
+
+In the meantime, here is a quick workaround:
+
+```sh
+podman machine init .... --image=docker://quay.io/podman/machine-os:5.2
+```
+
+## Smoother kind cluster experience
+
+Even if you do not have the `kind` CLI installed, Podman Desktop will guide you through the process of installing it when creating a Kind cluster,
+ensuring you can set up your cluster without leaving the workflow. No more diving back to shore for missing tools!
+
+<ThemedImage
+alt="Smoother kind cluster experience"
+sources={{
+    light: require('./img/podman-desktop-release-1.17/install-kind-cli-create-cluster-dark.png').default,
+    dark: require('./img/podman-desktop-release-1.17/install-kind-cli-create-cluster-dark.png').default,
+  }}
+/>
+
+### Kubernetes: Pods and CronJobs
+
+Since the last version, we have introduced a dedicated Kubernetes navigation bar. Now, we are making things even clearer:
+
+- Podman Pods and Kubernetes Pods are now separate.
```

Comment:
```suggestion
- [Podman Pods](https://docs.podman.io/en/stable/markdown/podman-pod.1.html) and [Kubernetes Pods](https://kubernetes.io/docs/concepts/workloads/pods/) are now separate.
```

I think it may be nice to provide a link for people curious to learn more about the difference between Podman pods and Kubernetes pods